### DISCUSSION TITLE
How to stop page reload errors (_error.jsx)

### DISCUSSION THREAD

**Discussion by youhosi** [vercel/next.js#10931] [Help]
Hello, is it possible to display _error.jsx without reloading the entire page? Apparently it has been patched, but I still have this problem.

**Comment by timneutkens** [comment]
Please provide a full reproduction.

**Comment by youhosi** [comment]
So yes, after switching to the home page (`inedx.jsx`) everything is displayed correctly. I have several paths in the application that are not yet created, after going to the link `/ contact` (the page does not exist yet) it takes me to the page error (`_error.jsx`). Unfortunately the entire page is then reloaded.

Video: https://streamable.com/wvwyc

**Reply by timneutkens** [reply]
Looks like expected behavior, if you go to a page and the link results in a 404 fetching the JavaScript for that page it will do a full page transition in order to make the zones feature possible (and this is generally more correct when incrementally adopting). It also really doesn't matter if it's a full page transition when going to 404 because 404 should be rare.

**Reply by youhosi** [reply]
@timneutkens Thanks for explanation! :)
